hexrd/ui/hexrd_config.py

```diff
@@ -1579,6 +1582,10 @@ def _set_materials(self, materials):
     def material(self, name):
         return self.config['materials']['materials'].get(name)
 
+    def check_activate_material_changed(self, material_name):
+        if material_name == self.active_material:
```

Comment:
This should be `self.active_material_name` instead (`self.active_material` returns the material object)

Reply:
Fixed